sdk-setup/src/proxymanager

```diff
@@ -0,0 +1,50 @@
+#!/bin/env python3
+#
+# Copyright (C) 2021 Jolla Ltd.
+# Contact: http://jolla.com/
+# All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions
+# are met:
+#
+# - Redistributions of source code must retain the above copyright
+#   notice, this list of conditions and the following disclaimer.
+# - Redistributions in binary form must reproduce the above copyright
+#   notice, this list of conditions and the following disclaimer in
+#   the documentation and/or other materials provided with the
+#   distribution.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+# AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+# ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+# LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+# CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+# SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+# INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+# CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+# ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+# POSSIBILITY OF SUCH DAMAGE.
+
+
+import dbus
+from dbus.mainloop.glib import DBusGMainLoop
+from gi.repository import GLib
+import json
+
+CONFIG_FILE = '/etc/mersdk/share/proxy.json'
+
+DBusGMainLoop(set_as_default=True)
+
+bus = dbus.SystemBus()
+manager_object = bus.get_object('org.pacrunner', '/org/pacrunner/manager')
+manager = dbus.Interface(manager_object, dbus_interface='org.pacrunner.Manager')
+
+with open(CONFIG_FILE, 'r') as proxy_json:
+    config = json.load(proxy_json)
+    manager.CreateProxyConfiguration(config)
+
```

Comment:
If I understood this correctly, we only read the config file on startup? Maybe we could also watch it for changes?

Reply:
With the original configuration we did it on startup and then on every connect. But I think it's better to avoid changes while the engine is (potentially) in use.